src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -503,36 +503,81 @@ public string DictionaryLink
         {
             get
             {
-                string finalLink = Configurations.DynamoDictionary + "#/";
-                if (category == null || category == "")
-                {
-                    return Configurations.DynamoDictionary; // if there is no category, return the link to home page
-                }
+                dictionaryLink = dictionaryLink ?? Configurations.DynamoDictionary;
+                return dictionaryLink;
+            }
+            set
+            {
+                dictionaryLink = value;
+            }
+        }
+
+        private string dictionaryLink;
+
+        public string GetDictionaryLinkFromLibrary(LibraryServices libraryServices)
+        {
+            string finalLink = Configurations.DynamoDictionary + "#/";
+            if (IsCustomFunction)
+            {
+                return ""; // If it is not a core or Revit function, do not display the dictionary link
+            }
+            if (category == null || category == "")
+            {
+                return Configurations.DynamoDictionary; // if there is no category, return the link to home page
+            }
+
+            int i = category.LastIndexOf(Configurations.CategoryDelimiterString);
+            switch (category.Substring(i + 1))
+            {
+                case Configurations.CategoryGroupAction:
+                    finalLink += ObtainURL(category.Substring(0, i));
+                    finalLink += "Action/";
+                    break;
+                case Configurations.CategoryGroupCreate:
+                    finalLink += ObtainURL(category.Substring(0, i));
+                    finalLink += "Create/";
+                    break;
+                case Configurations.CategoryGroupQuery:
+                    finalLink += ObtainURL(category.Substring(0, i));
+                    finalLink += "Query/";
+                    break;
+                default:
+                    finalLink += ObtainURL(category);
+                    finalLink += "Action/";
+                    break;
+            }
+            finalLink += this.ShortenName;
 
-                int i = category.LastIndexOf(Configurations.CategoryDelimiterString);
-                switch (category.Substring(i + 1))
+            // Check if the method has overloads
+            IEnumerable<FunctionDescriptor> descriptors = libraryServices.GetAllFunctionDescriptors(CreationName.Split('@')[0]);
+            if (descriptors != null && descriptors.Count() > 1)
```

Comment:
It's better to use `Any()` with `IEnumerable` so use `descriptors.Any()` instead of `Count()` as `Count()` will enumerate over the whole sequence.